package.json

```diff
@@ -25,7 +25,7 @@
     "fs-extra": "^11.1.1",
     "mocha": "^10.2.0",
     "nodemark": "^0.3.0",
-    "prebuild": "^13.0.0",
+    "prebuild": "^13.0.1",
```

Comment:
UGH prebuild version hygiene is deplorable. There isn't even a tag with this version number. What's in there? NO ONE KNOWS.